packages/nextjs/src/types.ts

```diff
@@ -0,0 +1,118 @@
+export { SDK_VERSION } from '@sentry/node';
+export { setContext } from '@sentry/node';
+export { setExtra } from '@sentry/node';
+export { setExtras } from '@sentry/node';
+export { setTag } from '@sentry/node';
+export { setTags } from '@sentry/node';
+export { setUser } from '@sentry/node';
+export { withScope } from '@sentry/node';
+export { NodeBackend } from '@sentry/node';
+export { NodeClient } from '@sentry/node';
+export { init } from '@sentry/node';
+export { Handlers } from '@sentry/node';
+export { BrowserClient } from '@sentry/react';
+export { injectReportDialog } from '@sentry/react';
+export { eventFromException } from '@sentry/react';
+export { eventFromMessage } from '@sentry/react';
+export { forceLoad } from '@sentry/react';
+export { onLoad } from '@sentry/react';
+export { showReportDialog } from '@sentry/react';
+export { wrap } from '@sentry/react';
+export { Profiler } from '@sentry/react';
+export { withProfiler } from '@sentry/react';
+export { useProfiler } from '@sentry/react';
+export { ErrorBoundary } from '@sentry/react';
+export { withErrorBoundary } from '@sentry/react';
+export { createReduxEnhancer } from '@sentry/react';
+export { reactRouterV3Instrumentation } from '@sentry/react';
+export { reactRouterV4Instrumentation } from '@sentry/react';
+export { reactRouterV5Instrumentation } from '@sentry/react';
+export { withSentryRouting } from '@sentry/react';
+
+// Merged arrays and objects
+
+// defaultIntegrations
+
+const nodeDefaultIntegrationsNames = [
+  'InboundFilters',
+  'FunctionToString',
+  'Console',
+  'Http',
+  'OnUncaughtException',
+  'OnUnhandledRejection',
+];
+const reactDefaultIntegrationsNames = ['TryCatch', 'Breadcrumbs', 'GlobalHandlers', 'UserAgent'];
+const nodeDefaultIntegrations = nodeSDK.defaultIntegrations.filter(
+  element => element.name in nodeDefaultIntegrationsNames,
+);
+const reactDefaultIntegrations = reactSDK.defaultIntegrations.filter(
+  element => element.name in reactDefaultIntegrationsNames,
+);
+export const defaultIntegrations = [...nodeDefaultIntegrations, ...reactDefaultIntegrations];
+
+// Transports
+
+const nodeTransportsNames = ['BaseTransport', 'HTTPTransport', 'HTTPSTransport'];
+const reactTransportsNames = ['FetchTransport', 'XHRTransport'];
+const nodeTransports = {} as { [key: string]: any };
+const reactTransports = {} as { [key: string]: any };
+nodeTransportsNames.forEach(elementName => {
+  nodeTransports[elementName] = nodeSDK.Transports[elementName as keyof typeof nodeSDK.Transports];
+});
+reactTransportsNames.forEach(elementName => {
+  reactTransports[elementName] = reactSDK.Transports[elementName as keyof typeof reactSDK.Transports];
+});
+export const Transports = { ...nodeTransports, ...reactTransports };
+
+// Integrations
+
+const nodeIntegrationsNames = [
+  'FunctionToString',
+  'InboundFilters',
+  'Console',
+  'Http',
+  'OnUncaughtException',
+  'OnUnhandledRejection',
+  'LinkedErrors',
```

Comment:
`LinkedErrors` integrations are different for node and browser. This will fail in React.